apps/src/lab2/types.ts

```diff
@@ -109,6 +109,8 @@ export interface VideoLevelData {
   download: string;
 }
 
+// TODO: Add AichatLevelData.
+
```

Comment:
Note that if AichatLevelData extends ProjectLevelData, it would probably be better to define the type within the aichat directory (like MusicLevelData, defined in music/types), so that the lab-specific level data definition can be closer to the lab that's using it

Reply:
Thanks for the feedback! I'll call this out in Slack for the fast-follow PRs, but I didn't move the comment for now, because I'm eager to get this into staging.